code-of-conduct.md

```diff
@@ -1,5 +1,3 @@
-## Kubernetes Community Code of Conduct
+# Kubernetes Community Code of Conduct
 
-Helm follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
-
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/code-of-conduct.md?pixel)]()
```

Comment:
This site is still live. Is this intentionally being phased out?

Reply:
I wasn't aware that it was still live, who can I talk to about whether these analytic links are actively being used and how/whether they should be put in place for each individual repo's .md files?

Reply:
`git blame` points the finger at @philips :)

https://github.com/kubernetes/helm/commit/c97f4d7cd36d07ff36ffd30d0df0739627a95e3e#diff-d3030a18b089fdb1fbfabf6e75e4aef0

@philips do you know if this is still in use?

Reply:
following the breadcrumbs brought me back to the original commit when the kubernetes community code of conduct was created, which still brings us back to Brendan. That's as far as my google-fu got me :) https://github.com/kubernetes/kubernetes-template-project/commit/25ead97bebdff65bfc177908ac79d4f00dd90bb6